elide-contrib/elide-dynamic-config-helpers/README.md

```diff
@@ -0,0 +1,19 @@
+## Validation for Dynamic Config
+
+Validate the config files in local before deployment.
+
+Sample command:
+```java
+Usage: java -cp <Jar File Name> com.yahoo.elide.contrib.dynamicconfighelpers.validator.DynamicConfigValidator <Path for Model Configs Directory>
+```
+Expected Model Configs Directory Structure:
+```java
```

Comment:
is any thing other than Java be used here?

Reply:
changed to text